tests/helpers.py

```diff
@@ -135,6 +136,10 @@ def create_pod(
         else:
             pod_data['spec']['volumes'][0]['persistentVolumeClaim']['claimName'] = pvc_name
 
+    if interface_type == constants.CEPHBLOCKPOOL and raw_block_pv:
+        pod_data['spec']['containers'][0]['volumeDevices'][0]['devicePath'] = '/dev/block'
```

Comment:
please save '/dev/block' as a constant in constants.py

Reply:
Udpated